### DISCUSSION TITLE
jq CLI

### DISCUSSION THREAD

**Discussion by yonas** [stephenberry/glaze#862] [Ideas]
Does a jq-like CLI exist that uses glaze?

**Comment by stephenberry** [comment]
Not at the moment. Glaze is getting more command line tools, a command line menu `glz::cli_menu` was recently added, and I plan to add an argument parser soon. A `jq-like` CLI would be cool, but I'll probably add more command line features to glaze before something like this so that everything works well together.

I'm going to move this into [Discussions](https://github.com/stephenberry/glaze/discussions) as an idea for the future, but it won't be a pressing issue to solve.
